Automattic/wp-calypso
# Congrats: remove legacy dependency code

Related to 6015-gh-Automattic/dotcom-forge

## Proposed Changes

Removes the now-unused "product detail" components that used to render on the legacy thank you pages

## Testing Instructions

The files removed were exporting components that are no longer utilized, so deep testing shouldn't be needed.

If you'd like to formally test, the best bet will be to make different types of purchases to confirm the congrats page still loads as expected.

Other than that we should just confirm that all tests still pass.
